<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinestraHome</class>
 <widget class="QMainWindow" name="FinestraHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>641</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>761</width>
    <height>641</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>761</width>
    <height>641</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>761</width>
     <height>641</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>761</width>
     <height>641</height>
    </size>
   </property>
   <widget class="QPushButton" name="ButtonListaClienti">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}
</string>
    </property>
    <property name="text">
     <string>Clienti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaOrdini">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}</string>
    </property>
    <property name="text">
     <string>Ordini</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaDipendenti">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}
</string>
    </property>
    <property name="text">
     <string>Dipendenti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaDocumenti">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}
</string>
    </property>
    <property name="text">
     <string>Documenti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaFornitori">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}

</string>
    </property>
    <property name="text">
     <string>Fornitori</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaEntiEsterni">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}

</string>
    </property>
    <property name="text">
     <string>Enti Esterni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaProdotti">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}

</string>
    </property>
    <property name="text">
     <string>Prodotti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonListaMacchine">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #283747;
    border-radius: 8px;
    border: 3px solid #FFFFFF; /* Bordo bianco esterno */
    box-sizing: border-box;
    color: #FFFFFF;
    padding: 8px;
    text-align: center;
    display: inline-block;

    font-size: 20px;
    font-weight: 500;
    line-height: 1.5;
    margin: 10px 2px;
    cursor: pointer;
    transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out;   

}

QPushButton:hover {
    background-color: #1E2230;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al passaggio del mouse */
}

QPushButton:focus {
    background-color: #242835;
    border-color: #FFFFFF; /* Mantieni il bordo bianco al focus */
    box-shadow: rgba(255, 255, 255, 0.15) 0 1px 0 inset, rgba(46, 54, 80, 0.075) 0 1px 1px, rgba(104, 101, 235, 0.5) 0 0 0 32px;
}

</string>
    </property>
    <property name="text">
     <string>Macchine</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSfondoFinestraHome">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>641</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/immaginifinestrahome/25101_XCDaZO);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../../../../Program Files (x86)/Qt Designer/img.qrc">:/immaginifinestrahome/25101_XCDaZO</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="lblDataTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'Montserrat', sans-serif;
font-size: 10px;
font-weight: 600;
line-height: 1.2;
letter-spacing: 1px;
text-transform: uppercase;
color: #333;
text-align: right;

background-color: transparent;
color: white; /* Cambia il colore a uno chiaro */

</string>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>9</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
   </widget>
   <widget class="QLabel" name="BottonTendina">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/immaginifinestrahome/9042808_menu_icon.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/immaginihome/9042808_menu_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BottonInfoSoftware">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/immaginifinestrahome/9042463_language_icon.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/immaginihome/9042880_profile_circled_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BottonUtenteConnesso">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/immaginifinestrahome/9042880_profile_circled_icon.svg);
background-color: transparent;
color: white; /* Cambia il colore a uno chiaro */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/immaginihome/9042463_language_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="BarraAlta">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #283747; /* Blu scuro */
border-bottom: 2px solid white; /* bordo bianco inferiore */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>labelSfondoFinestraHome</zorder>
   <zorder>BarraAlta</zorder>
   <zorder>ButtonListaClienti</zorder>
   <zorder>ButtonListaDocumenti</zorder>
   <zorder>ButtonListaEntiEsterni</zorder>
   <zorder>ButtonListaMacchine</zorder>
   <zorder>ButtonListaFornitori</zorder>
   <zorder>ButtonListaOrdini</zorder>
   <zorder>ButtonListaProdotti</zorder>
   <zorder>ButtonListaDipendenti</zorder>
   <zorder>BottonTendina</zorder>
   <zorder>BottonInfoSoftware</zorder>
   <zorder>BottonUtenteConnesso</zorder>
   <zorder>lblDataTime</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../Program Files (x86)/Qt Designer/img.qrc"/>
 </resources>
 <connections/>
</ui>
